adaptdl/adaptdl/speedup.py

```diff
@@ -53,15 +53,23 @@ class SpeedupFunction(object):
 
     def __init__(self, params, grad_params=None, init_batch_size=None,
                  max_batch_size=None, local_bsz_bounds=None,
-                 elastic_bsz=False):
+                 grad_acc_steps=None, elastic_bsz=False):
```

Comment:
`grad_acc_steps` should be an internal detail of the SpeedupFunction, i.e. not an argument of `__init__` but an argument of `_goodput` instead.

Reply:
This is just a setting for the maximum number of sets. I think it fits in logically better with the other maximum specifications. Maybe this should be renamed tho to indicate better that it's a maximum.

Reply:
Changed to be a boolean flag on initialization to be whether to allow or disallow gradient accumulation.